schema/validate.go

```diff
@@ -19,12 +19,12 @@ func main() {
 
 	schemaPath := os.Args[1]
 	if !strings.Contains(schemaPath, "://") {
-		schemaPath, err := filepath.Abs(schemaPath)
+		absPath, err := filepath.Abs(schemaPath)
```

Comment:
Obsolete now that #552 has landed [with this change][1]?

[1]: https://github.com/opencontainers/runtime-spec/pull/552/files#diff-8ce800f4e5ad55c2ec4bdcd49613f4c9R44